src/Themes/Default/SearchPromptRenderer.php

```
@@ -123,6 +128,21 @@ protected function renderOptions(SearchPrompt $prompt): string
         ));
     }
 
+    /**
+     * Calculate the width for description text.
+     */
+    protected function calculateDescriptionWidth(Prompt $prompt, int $maxWidth): int
```

PHPMD (2.15.0)

Avoid unused parameters such as '$maxWidth'. (Unused Code Rules)

(UnusedFormalParameter)